### ISSUE TITLE
Creating a filtered dataset generates error

### ISSUE THREAD

**Issue by qo4on** [vaexio/vaex#417]
This code from your official tutorial shows error in Colab:
```
df_negative = df[df.x < 0]
df_negative[['x', 'y', 'z', 'r']]
```
> ---------------------------------------------------------------------------
> ValueError                                Traceback (most recent call last)
> /usr/local/lib/python3.6/dist-packages/IPython/core/formatters.py in __call__(self, obj, include, exclude)
>     968                 d['include'] = include
>     969                 d['exclude'] = exclude
> --> 970                 return method(**d)
>     971             return None
>     972         else:
> 
> 6 frames
> /usr/local/lib/python3.6/dist-packages/vaex/dataframe.py in copy(self, column_names, virtual)
>    4591                     df._selection_masks[key] = vaex.superutils.Mask(df._length_original)
>    4592                 # and make sure the mask is consistent with the cache chunks
> -> 4593                 np.asarray(df._selection_masks[key])[:] = np.asarray(self._selection_masks[key])
>    4594         for key, value in self.selection_history_indices.items():
>    4595             if self.get_selection(key):
> 
> ValueError: could not broadcast input array from shape (330000) into shape (5)
> ---------------------------------------------------------------------------
> TypeError                                 Traceback (most recent call last)
> <ipython-input-24-aeaf77ba01d1> in <module>()
>       1 df_negative = df[df.x < 0]
> ----> 2 df_negative[['x', 'y', 'z', 'r']]
> 
> 2 frames
> /usr/local/lib/python3.6/dist-packages/IPython/core/formatters.py in format(self, obj, include, exclude)
>     150             return {}, {}
>     151 
> --> 152         format_dict, md_dict = self.mimebundle_formatter(obj, include=include, exclude=exclude)
>     153 
>     154         if format_dict or md_dict:
> 
> TypeError: 'NoneType' object is not iterable

**Comment by JovanVeljanoski** [comment]
Hi @qo4on 

Thanks for trying out vaex. Unfortunately, I can not reproduce your issue.
Few things to try:
- make sure the `df = vaex.example()` downloaded correctly (it may take some time in colab)
- how do you installed vaex? `pip install vaex` usually works for me, but sometimes requires restart of the runtime, i do not understand why
- maybe you can share a bit more of the notebook code, so I can try to reproduce the issue?

This is how it looks like for me:
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/18574951/64926047-32c34f80-d7f9-11e9-8249-edf1e5829bd6.png">

**Comment by qo4on** [comment]
Thank you for your answer, JovanVeljanoski.
To reproduse this error please run this code with Python 3 GPU runtime in Colab:
https://drive.google.com/open?id=14v3mTDLq4I_8fXBuwXh9Fzj-3CcsmLw_

Also I had troubles with installing tensorflow 2.0 RC1 after importing vaex recently. But now it works.

**Comment by qo4on** [comment]
Google drive shows 404 error for the link above. I don't know why. The same file in colab: https://colab.research.google.com/drive/14v3mTDLq4I_8fXBuwXh9Fzj-3CcsmLw_

**Comment by qo4on** [comment]
I found the issue. When you run
```
df_negative = df[df.x < 0]
df_negative[['x', 'y', 'z', 'r']]
```
it works. But if you run the same code after selection it doesn't:
```
df.select(df.x < 0)
df.evaluate(df.x, selection=True)
df_negative = df[df.x < 0]
df_negative[['x', 'y', 'z', 'r']]
```

**Comment by JovanVeljanoski** [comment]
Hi, 

Thanks for tracking this down. I managed to reproduce it and make a unit-test. Looks like a small bug indeed.

Note that the issue appears only when you try to display/print the filtered dataframe.

If you do something like:
```
df_negative = df[df.x < 0]
df_negative['x'] 
df_negative.plot1d('x')
```
things will still work. The issue is only with the displaying of the dataframe (as far as I can tell with a very quick review).

**Comment by qo4on** [comment]
There are other errors in your official tutorial. Run all cells from [here ](https://github.com/vaexio/vaex/blob/master/docs/source/tutorial.ipynb) in colab.

**Comment by maartenbreddels** [comment]
This is fixed, and released (vaex core 1.1 or vaex 2.2.1), thanks for the report!
